fb_dnsmasq/recipes/default.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Cookbook Name:: fb_dnsmasq
+# Recipe:: default
+#
+# Copyright (c) 2016-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+package 'dnsmasq' do
+  action :upgrade
+end
+
+template '/etc/ethers' do
```

Comment:
Ah interesting, I did not know this was a general purpose thing, but you're right. I'll spin this out into a `fb_ethers` cookbook.